release-notes/6.0/supported-os.json

```diff
@@ -249,7 +249,7 @@
                         "7"
                     ],
                     "notes": [
-                        "Red Hat family distributions are supported per [Linux compatibility and support](../../linux-support.md)."
+                        "RHEL-compatible distributions are supported per [.NET Support](../../support.md)."
```

Comment:
I don't think I see anything in support.md mentioning "RHEL-compatible" support. Should that broad of a statement be clearly defined?

Reply:
Great point. This is what I intend folks to look at.

https://github.com/dotnet/core/pull/9402/files#diff-017fcd31229e5b6d295ec48a4746aa15fb14eaf39ed3213ebb0e10436c73a90cR19

Three ideas:

- Use the term "RHEL-compatible derivatives" instead of "RHEL-compatible distributions"
- Use the term "Compatible derivatives" instead of "Downstream derivative support"
- Do both.

Reply:
Made those changes.